extensions/wikia/SiteWideMessages/templates/editor.tmpl.php

```diff
@@ -163,6 +163,34 @@
 								<?= wfMsg('swm-label-mode-users-group-hint') ?>
 							</td>
 						</tr>
+						<tr>
+							<td>
+								<input name="mSendModeUsers" id="mSendModeUsersPU" type="radio" value="POWERUSER"<?= $formData['sendModeUsers'] == 'POWERUSER' ? ' checked="checked"' : ''?>/>
+							</td>
+							<td>
+								<label for="mSendModeUsersPU"><?= wfMessage( 'swm-label-mode-users-poweruser' )->escaped() ?></label>
+							</td>
+							<td>
+								<?php
+									foreach	( $formData[ 'powerUserTypes' ] as $powerUserType ) {
+										$label = '<label for="mPowerUserType_' . Sanitizer::encodeAttribute( $powerUserType ) . '">';
+											$label .= Sanitizer::escapeHtmlAllowEntities( $powerUserType ) . '</label><br>';
+										$html = '<input type="checkbox" name="mPowerUserType[]"';
+											$html .= 'id="mPowerUserType_' . Sanitizer::encodeAttribute( $powerUserType ) . '"';
+											$html .= 'value="' . Sanitizer::encodeAttribute( $powerUserType ) . '"';
+										if ( isset( $formData['mPowerUserType'] ) && in_array( $powerUserType, $formData['mPowerUserType'] ) ) {
+											$html .= ' checked="checked"';
+										}
+										$html .= '>';
+										echo $html;
```

Comment:
In general, it's always better to wrap a checkbox in the label so there isn't an area between the checkbox and the text that can't be clicked.